ru-ru/d-ru.html.markdown

```diff
@@ -0,0 +1,735 @@
+---
+language: d
+filename: learnd-ru.d
+contributors:
+    - ["Anton Pastukhov", "http://dprogramming.ru/"]
```

Comment:
You should place your name to the `translators` section (see, for example, Russian article for Python or Perl)